build/custom/phing/build.xml

```diff
@@ -3,7 +3,7 @@
 <project name="blt" default="list">
 
   <!-- Define blt root directory. -->
-  <property name="blt.root.relative" value="${project.basedir}/../.."/>
+  <property name="blt.root.relative" value="${project.basedir}/../../.."/>
```

Comment:
Right now there are 2 instances of blt.sh which are the same minus the path they call. This means duplicated efforts when making changes to the script. This changes them to both rely on the same patch structure, and the initial create copies the existing blt.sh into the target directory.